src/gpt-auto-generator/gpt-auto-generator.c

```diff
@@ -612,7 +612,7 @@ static int add_root_mount(void) {
 
         r = efi_loader_get_device_part_uuid(NULL);
         if (r == -ENOENT) {
-                log_debug("EFI loader partition unknown, exiting.");
+                log_notice("EFI loader partition unknown, exiting.\n(The boot loader did not set LoaderDevicePartUUID.)");
```

Comment:
maybe say explicitly that LoaderDevicePartUUID is an EFI variable. People might know be aware of the concept, they need something to google.